k8s/upgrades/0.9.0-1.0.0/jiva/jiva_volume_upgrade_auto.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+source ../util.sh
```

Comment:
I think it has tight dependency with scripts where we are running. Since we are making it generic we can go with
```sh
SOURCE="${BASH_SOURCE[0]}"
while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
DIR="$( cd -P "$( dirname "$SOURCE" )/../" && pwd )"
source $DIR/util.sh
```

Reply:
Done